tests/conftest.py

```diff
@@ -2773,6 +2770,7 @@ def rnafusion_context(
         status_db,
         internal_id=sample_id,
         reads_updated_at=datetime.now(),
+        reads=200000000,
```

Comment:
```suggestion
        reads=200_000_000,
```
Perhaps make a fixture instead of hard coding?